src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
```

Comment:
PullProofAI

The @IsArray() validation decorator has been added to ensure 'practices' is an array.
```diff
+   @IsArray()
```


<!-- This is an auto-generated comment by PullProofAI -->